### ISSUE TITLE
VoiceOver wrong focus order - accessibility

### ISSUE THREAD

**Issue by llune** [nextcloud/ios#743]
### Expected behaviour
The focus with VoiceOver should be the following.

1. NavigationMenuController: the label should be explicit like Open menu.
2. Focus on the expanded items.
3. When on the last item next focus should go back on the Menu button and the label should be Close Menu.
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
### Actual behaviour                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
The current behaviour is not good. When the menu is expanded the focus is made on the items behind the menu (Search, the files) first.

### Steps to reproduce                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
Here's a video of the current VoiceOver focus : [nextcloud-ios.mov.zip](https://github.com/nextcloud/ios/files/2626558/nextcloud-ios.mov.zip)

### iOS version
12.1

### App version
2.22.5

**Comment by bdorer** [comment]
this ticket is still valid for the file context menu. If you open it, you are able to close it by doing the VO escape gesture. Another way is to swipe to the end of the menu and VO says for German "Kartenverwaltung" in english VO says probably "card management". If you swipe down here, there's the action to cancel the menu. This are two unneeded extra steps for VO users.
I use the latest app store version.
